capstone/src/components/User/Adventurer/ExperienceInfo/ExperienceInfo.jsx

```diff
@@ -25,6 +51,42 @@ export default function ExperienceInfo({ restaurants }) {
     ? getPhotoReference(currentRestaurant)
     : null;
 
+  // get active feedback information
+  useEffect(() => {
+    if (currentRestaurant) {
+      getActiveFeedback(currentRestaurant.place_id).then((res) =>
+        setActiveFeedback(res.data),
```

Comment:
Instead of initializing the reviews below in the use effect, it'd be better to wait until both these requests finish (using Promise.all for example) and initialize it afterward.